update organizations page (#47)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/admin/organizations.mdx

| Key path                                               | Type    | Default | Description                                                                                                                                                                                                                                                                                         |
| ------------------------------------------------------ | ------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `data.type`                                            | string  |         | Must be `"organizations"`                                                                                                                                                                                                                                                                           |

> | `data.attributes.access-beta-tools`                    | boolean | false   | Whether or not workspaces in the organization can be configured to use beta versions of Terraform.                                                                                                                                                                                                  |

Reviewer: ```suggestion
| `data.attributes.access-beta-tools`                    | boolean | false   | Whether or not workspaces in the organization can be configured to use beta Terraform versions.                                                                                                                                                                                                  |
```

| `data.attributes.global-module-sharing`                | boolean | false   | If true, modules in the organization's private module repository will be available to all other organizations in this TFE instance. Enabling this will disable any previously configured [module consumers](#list-module-consumers-for-an-organization).                                            |

> | `data.attributes.is-disabled`                          | boolean | false   | Disabling the organization will remove all permissions and no longer be accessible to users.                                                                                                                                                                                                        |

Reviewer: ```suggestion
| `data.attributes.is-disabled`                          | boolean | false   | Removes all permissions from the organization and makes it inaccessible to users.                                                                                                                                                                                                        |
```
